### PR TITLE
Evn

### PR DESCRIPTION
**Pull request by aleg47**
Update for EVN model ZBHS4RGBW and add device ZB24100VS

### CODE REVIEW

File: devices/evn.js [hunk 1]
```diff
@@ -1,26 +1,31 @@
...
-            'color_move', 'hue_stop'])],
+        fromZigbee: [fz.battery, fz.command_move_to_color, fz.command_move_to_color_temp, fz.command_move_hue,
+            fz.command_step, fz.command_stop, fz.command_move, fz.command_recall, fz.command_on, fz.command_off],
+        exposes: [e.battery(), e.action([
+            'color_move', 'color_temperature_move', 'brightness_step_up', 'brightness_step_down',
+            'brightness_move_up', 'brightness_move_down', 'brightness_stop',
+            'hue_move', 'hue_stop', 'recall_*', 'on', 'off'])],
         toZigbee: [],
         meta: {multiEndpoint: true, battery: {dontDividePercentage: true}},
-        configure: async (device, coordinatorEndpoint) => {
```
Comment (Koenkk) [inline devices/evn.js:19]: Does it still report battery after this is removed? Repair the device and check if it still does.

**Comment by Koenkk** [comment]
Note to self; after this merge https://github.com/Koenkk/zigbee2mqtt.io/pull/1296/files

File: devices/evn.js (hunk 1 above)
Comment (aleg47) [reply devices/evn.js:19]: I am not so familiar with the reporting mechanism. But after switching on the device, the following message appears:

`Received Zigbee message from 'RGBW Fernbedienung', type 'attributeReport', cluster 'genPowerCfg', data '{"batteryPercentageRemaining":100}' from endpoint 1 with groupID null`

After that, this message comes cyclically:

`MQTT publish: topic 'zigbee2mqtt/RGBW Fernbedienung', payload '{"battery":100,"last_seen":1648152834008,"linkquality":111,"voltage":3800}'`

**Comment by Koenkk** [comment]
thanks!

**Pull request merged**